tox.ini

```diff
@@ -1,40 +1,42 @@
 [tox]
-envlist = py{36,37}-{offline,astropydev,numpydev,build_docs},conda,pycodestyle
+envlist = py{36,37}-{offline,astropydev,numpydev,sunpydev},build_docs,conda,pycodestyle
 
 [testenv]
-isolated_build=true
+changedir = tmp
 setenv =
     MPLBACKEND = agg
     COLUMNS = 180
+    PYTEST_COMMAND = pytest --pyargs ndcube --cov=ndcube --cov-config={toxinidir}/setup.cfg --verbose --durations=25
```

Comment:
We don't need to do pyargs here as there are no C extensions.

Reply:
Is there any harm in leaving it? What if C extensions are added in the future?

Reply:
well it means you can't run part of the tests with tox.

Reply:
Updated it.